config.md

```diff
@@ -324,15 +324,24 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 **`annotations`** (object, optional) contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be a key-value map where both the key and value MUST be strings.
+While the value MUST be present, it MAY be an empty string.
+Keys MUST be unique within this map, and best practice is to namespace the keys.
+Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
+Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by subsequent specifications.
```

Comment:
Missing the trailing dot for `org.opencontainers.`.

“subsequent specifications” is unclear to me.  Can we [follow image-spec][1] and use “… are reserved for future versions of the specification.”?

[1]: https://github.com/opencontainers/image-spec/blame/08ef0b1d662ac889d3bf2e99957acddad8e5804f/descriptor.md#L37